src/els_distribution_server.erl

```diff
@@ -168,10 +168,9 @@ ensure_epmd() ->
   0 = els_utils:cmd("epmd", ["-daemon"]),
   ok.
 
--spec node_name(string(), binary()) -> atom().
-node_name(Prefix, Binary) ->
-  <<SHA:160/integer>> = crypto:hash(sha, Binary),
-  Int = erlang:unique_integer([positive]),
-  Id = lists:flatten(io_lib:format("~40.16.0b_~p", [SHA, Int])),
+-spec node_name(binary(), binary()) -> atom().
+node_name(Prefix, Name) ->
+  Int = erlang:phash2(erlang:timestamp()),
```

Comment:
If we used a string representation instead of the phash2, either as a time or of the underlying integers, we could see which process was "older". Or is it some meaningless large negative number?

Reply:
The `monotonic_time` was returning a negative number, that's why I switched for this. `erlang:timestamp/0` returns a tuple. The "age" of a process should still be visible via `ps`. Also, I suspect the port numbers displayed by `epmd -names` are somewhat incremental.

Reply:
ok, happy with this as is then.